mobile/src/screens/components/TipCard.tsx

```diff
@@ -44,18 +45,26 @@ function getIconTitle(item: ITipItem) {
 export default function TipItemCard(props: { item: ITipItem, onPress: () => void }) {
     const { onPress, item } = props;
     const { type, title } = item;
+    const [loading, setLoading] = useState(true);
+    const data = useHealthKit();
+
+    useEffect(() => {
+        if (data) setLoading(false);
+      }, [data]);
 
     const isExternal = (item.type === 'staticTip' || item.type === 'docLinkTip') && !!item.url;
 
+
     return (
         <TouchableOpacity onPress={onPress}>
             <View style={[styles.card]}>
                 <Text
                     numberOfLines={2}
                     style={[!Layout.isSmallDevice ? TextStyles.p1 : TextStyles.p3, styles.cardTitle]}
-                >
+                > 
                     {title}
                 </Text>
+                <Text style={{color: "white"}}>Gender: {loading ? "Fetching" : data.value}</Text>
```

Comment:
We should guard against errors and undefined data here

Reply:
Right now it's protecting against loading errors and if the gender is not set in healthkit, "unknown" will be displayed. I'm thinking we should wait on adding more guards until we have more information on our use cases (what exact data we want to retrieve and how we'll use the data).

Adding this code in the TipCard was just to test the connection was working. I'm going to remove it before it's merged. Let me know what you think.

Reply:
Yup! - Wanted to confirm that unknown was the catch-all case, and not a specific value. Sounds good to me